File: LAB1/in.c
#include<stdio.h>
#include<stdlib.h>
int main(){
	FILE *in,*out;
	in=fopen("in.txt","r");
	out=fopen("/home/student/Documents/ou.txt","w");
	if(in==NULL || out==NULL){
		printf("Error! Could not open file\n"); 
              exit(-1);
          }
         char c;
         c=fgetc(in);
         while(c!=EOF){
         	fputc(c,out);
         	c=fgetc(in);
         }
         fclose(in);
         fclose(out);
         return 0;

}

File: LAB1/in.java
import java.io.*;
public class in{
	public static void main(String [] args){
		FileReader in=null;
		FileWriter out=null;
		try{
			in=new FileReader("in.txt");
			out=new FileWriter("/home/student/Documents/out.txt");
			int c;
			while((c=in.read())!=-1){
				out.write(c);
			}
		}catch(Exception e){

		}
		finally{
			try{
			if(in!=null) {

				in.close();
				
			}

			if(out!=null)
				{ 
					out.close();
			}}catch(Exception e){
				
			}
		}

	}
}